select.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto2";
+
+package tipb;
+
+option java_multiple_files = true;
+option java_package = "com.pingcap.tidb.tipb";
+option java_outer_classname = "SelectProto";
+
+import "expression.proto";
+import "schema.proto";
+import "error.proto";
+
+// HandleRange specifies the range of handle to be scanned.
+// low is closed, high is open ( low <= value < high ).
+message HandleRange {
+	optional int64 low = 1;
+	optional int64 high = 2;
+}
+
+// SelectRequest works like a simplified select statement.
+message SelectRequest {
+	// transaction start timestamp.
+	optional int64 start_ts = 1;
```

Comment:
Can we use ts instead, seems kv need not know the transaction, only using in MVCC?